.github/workflows/make-release.yml

```diff
@@ -19,26 +21,45 @@ jobs:
         ref: ${{ github.ref }}
```

Comment:
Do you really need to keep the `ref` argument?

Reply:
Yes ! Sometimes, the checkout is not properly done. It happened that I did not get the modifications from my PR into the resulting package...